static/gsApp/components/addEventsCTA.tsx

```diff
@@ -26,10 +27,23 @@ import {openOnDemandBudgetEditModal} from 'getsentry/views/onDemandBudgets/editO
  */
 export type EventType = {
   [K in keyof typeof DATA_CATEGORY_INFO]: (typeof DATA_CATEGORY_INFO)[K]['isBilledCategory'] extends true
-    ? (typeof DATA_CATEGORY_INFO)[K]['name']
+    ? (typeof DATA_CATEGORY_INFO)[K]['singular']
```

Comment:
i introduced `singular` purely because of `EventType`

If I instead let it use the new `DataCategoryExact` values, any customers that muted the nav billing status alert will have it effectively unmuted since we check for mutes by looking in local storage using the old DataCategoryExact values as the key